app/cdap/text/text-en.yaml

```diff
@@ -2088,7 +2088,10 @@ features:
         distributed: Distributed
         localSandbox: Local Sandbox
     rulesmgmt: Rules
-
+  NoRecordScreen:
```

Comment:
Please add strings under `DataPrep`: https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/text/text-en.yaml#L463
or create a new top-level entry for the new Wrangler UI

Reply:
Actually, let's create a new top-level header for the new UI

Reply:
Done. created top-level-header(WranglerNewUI)